README.md

```diff
@@ -547,6 +547,23 @@ const uiSchema = {
 };
 ```
 
+Unlike other types, array's do not support the `required` property.  You can however instead specify the minimum number of items the user must select with the `minItems` property.
```

Comment:
It actually supports the `required` property, it only checks that an array property value is provided, which may or may not be empty. Put another way, for

```js
schema = {
  type: "object",
  required: ["foo"],
  properties: {
    foo: {type: "array"}
  }
}
```

`{foo: []}` is valid against this schema while `{}` is not.